src/rdkafka.h

```diff
@@ -9850,6 +9906,131 @@ RD_EXPORT void rd_kafka_DeleteAcls(rd_kafka_t *rk,
 
 /**@}*/
 
+/**
+ * @name Admin API - Elect Leaders
+ * @{
+ *
+ *
+ *
+ */
+
+/**
+ * @brief Start leader election for the specified Topic Partitions.
+ */
+typedef struct rd_kafka_ElectLeaders_s rd_kafka_ElectLeaders_t;
+
+/**
+ * @enum rd_kafka_ElectionType_t
+ * @brief Apache Kafka Election Types
+ */
+typedef enum rd_kafka_ElectionType_t {
+        RD_KAFKA_ELECTION_TYPE_PREFERRED = 0, /**< Preferred Replica Election */
+        RD_KAFKA_ELECTION_TYPE_UNCLEAN   = 1  /**< Unclean Election */
+} rd_kafka_ElectionType_t;
+
+/**
+ * @brief Create a new elect leaders object. This object is later passed to
```

Comment:
This should be done to new and destroy methods. Same with ElectLeadersResult as well.

```suggestion
 * @brief Create a new rd_kafka_ElectLeaders_t object. This object is later passed to
```